frontend/src/uxbox/main/ui/workspace/sidebar/options/circle.cljs

```diff
@@ -10,14 +10,16 @@
 (ns uxbox.main.ui.workspace.sidebar.options.circle
   (:require
    [rumext.alpha :as mf]
-   [uxbox.main.ui.workspace.sidebar.options.fill :refer [fill-menu]]
+   [uxbox.main.ui.workspace.sidebar.options.fill :refer [fill-attrs fill-menu]]
    [uxbox.main.ui.workspace.sidebar.options.measures :refer [measures-menu]]
-   [uxbox.main.ui.workspace.sidebar.options.stroke :refer [stroke-menu]]))
+   [uxbox.main.ui.workspace.sidebar.options.stroke :refer [stroke-attrs stroke-menu]]))
 
 (mf/defc options
   [{:keys [shape] :as props}]
-  [:div
-   [:& measures-menu {:shape shape
-                      :options #{:size :position :rotation}}]
-   [:& fill-menu {:shape shape}]
-   [:& stroke-menu {:shape shape}]])
+  (let [ids [(:id shape)]
+        stroke-values (select-keys shape stroke-attrs)]
+    [:div
```

Comment:
this `div` is necessary? if not, you can replace it with `[:*` that uses React fragment and does not creates additional DOM nesting.

Reply:
This was here before, but you are true. Changed in all shapes.